transactions/urls.py

```diff
@@ -23,5 +23,6 @@
 
 urlpatterns = [
     path('fundTransfer', views.fundTransfer, name='fund-transfer'),
-    path('pendingTransactions', views.pendingTrans, name='pending-transactions')
+    path('pendingTransactions', views.pendingTrans, name='pending-transactions'),
+    path('updateTransaction', views.updateTransaction, name='update-transaction')
```

Comment:
Like we planned, let's move both viewing pending requests and their approvals to inside internal_users app.

Reply:
Let's discuss about this, I still have some doubts about these features.